### PR TITLE
JWT token signed with RS256 for 3.1

### PR DESCRIPTION
**Pull request by korycins**
I want to merge this change because it is a port of changes for #7990

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-5833 Sign JWT tokens with RS256 instead of HS256](https://app.clickup.com/t/2549495/SALEOR-5833)

**Review by IKarbowiak — APPROVED** [review]

File: .isort.cfg [hunk 1]
```diff
@@ -1,2 +1,6 @@
 [settings]
-known_third_party = Adyen,PIL,authorizenet,babel,boto3,botocore,braintree,celery,dateutil,dj_database_url,dj_email_url,django,django_cache_url,django_countries,django_filters,django_measurement,django_prices,django_prices_openexchangerates,django_prices_vatlayer,draftjs_sanitizer,faker,freezegun,google,google_measurement_protocol,graphene,graphene_django,graphene_federation,graphql,graphql_relay,html2text,html_to_draftjs,i18naddress,jaeger_client,jwt,kombu,lxml,markdown,measurement,micawber,mptt,oauthlib,openpyxl,opentracing,petl,phonenumber_field,phonenumbers,pkg_resources,prices,promise,pybars,pytest,pythonjsonlogger,pytimeparse,pytz,razorpay,requests,sendgrid,sentry_sdk,storages,stripe,urllib3,uvicorn,versatileimagefield,weasyprint
+<<<<<<< HEAD
```
Comment (fowczarek) [inline .isort.cfg:2]: :policeman:  Not resolved conflicts.

Comment (fowczarek) [inline .isort.cfg:2]: :policeman:  Not resolved conflicts.

Comment (korycins) [reply .isort.cfg:2]: thanks for catching this

File: CHANGELOG.md [hunk 2]
```diff
@@ -316,6 +316,8 @@ All notable, unreleased changes to this project will be documented in this file.
   and `CheckoutPricesData` instead of `PricesData`
 - Do no allow using `id` for updating checkout and order metadata - #8906 by @IKarbowiak
   - Use `token` instead
+- Add additional validation for `from_global_id_or_error` function - #8780 by @CossackDex
```
Comment (fowczarek) [inline CHANGELOG.md]: Is it a proper changelog entry?

**Review by fowczarek — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (fowczarek) [reply CHANGELOG.md]: @korycins faster than his shadow :+1:

**Review by eCzerniak — APPROVED** [review]

**Pull request merged**